docs/releases/1.10.txt

```diff
@@ -212,6 +212,16 @@ CSRF
   accepts an optional ``template_name`` parameter, defaulting to
   ``'403_csrf.html'``, to control the template used to render the page.
 
+* To protect against `BREACH`_ attacks, the CSRF protection scheme is changed
```

Comment:
simplified along these lines ("the CSRF protection mechanism now  changes the form token")